lib/features/home/presentation/screens/home_screen.dart

```diff
@@ -0,0 +1,245 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+import '../../domain/entities/event_entity.dart';
+import '../cubit/home_cubit.dart';
+import '../widgets/event_large_cell.dart';
+import '../widgets/event_list_cell.dart';
+
+class HomeScreen extends StatefulWidget {
+  const HomeScreen({Key? key}) : super(key: key);
+
+  @override
+  State<HomeScreen> createState() => _HomeScreenState();
+}
+
+class _HomeScreenState extends State<HomeScreen> {
+  _getHomeData() => context.read<HomeCubit>().getHomeData();
```

Comment:
logic need to be called on the initialization for the cubit not from widget

Reply:
Done